resources/scss/components/_icons.scss

```diff
@@ -17,3 +17,11 @@
         transform: rotate(90deg);
     }
 }
+
+.right-triangle {
```

Comment:
Is this needed even though it looks like an SVG was added: `svg/video-play.blade.php` ?